src/renderers/shared/reconciler/ReactCompositeComponent.js

```diff
@@ -498,6 +498,7 @@ var ReactCompositeComponentMixin = {
     var inst = this._instance;
     var childContext = inst.getChildContext && inst.getChildContext();
     if (childContext) {
+      ReactCurrentOwner.processingChildContext = true;
```

Comment:
Are there any examples of the Devtool api used in React code?  
Would I be writing my own devtool function or do I use something like `ReactDOMInstrumentation.debugTool.onSetValueForProperty(inst, 'processingChildContext', true)`